### PR TITLE
Added Additional Info About Billing Agreements

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
## Purpose of this pull request

Added Additional Info About Billing Agreements

## Affected documentation pages

https://docs.magento.com/user-guide/sales/billing-agreements.html

whatsnew
Updated the [Billing Agreements](https://docs.magento.com/user-guide/sales/billing-agreements.html) topic to include detailed information about managing billing agreements.

### CODE REVIEW

File: src/sales/billing-agreements.md [hunk 1]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. Find the billing agreement in the list.
+
+1. Click the needed billing agreement to be viewed.
+
+1. In the right upper corner click the **Delete** button.
+
+1. To confirm the action click **OK**.
+
+### Grid Column descriptions
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:68]:
```suggestion
## Column descriptions
```

File: src/sales/billing-agreements.md [hunk 2]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. To confirm the action click **OK**.
+
+## Delete the Billing Agreement
+
+1. On the _Admin_ sidebar, go to  **Sales** > _Operations_ > **Billing Agreements**.
+
+1. Find the billing agreement in the list.
+
+1. Click the needed billing agreement to be viewed.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:62]:
```suggestion
1. Find the billing agreement in the list and click to open it.
```

File: src/sales/billing-agreements.md [hunk 3]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. In the right upper corner click the **Delete** button.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:64]:
```suggestion
1. In the top-right corner, click **Delete**.
```

File: src/sales/billing-agreements.md [hunk 4]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. To confirm the action click **OK**.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:66]:
```suggestion
1. To confirm the action, click **OK**.
```

File: src/sales/billing-agreements.md [hunk 5]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+![Billing Agreement View]({% link images/images/sales-billing-agreement-related-orders-tab.png %}){: .zoom}
+_Billing agreement related orders tab_
+
+## Cancel the Billing Agreement
+
+1. On the _Admin_ sidebar, go to  **Sales** > _Operations_ > **Billing Agreements**.
+
+1. Find the billing agreement in the list.
+
+1. Click the needed billing agreement to be viewed.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:50]:
```suggestion
1. Find the billing agreement in the list and click to open it.
```

File: src/sales/billing-agreements.md [hunk 6]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. In the right upper corner click the **Cancel** button.
+
+1. To confirm the action click **OK**.
+
+## Delete the Billing Agreement
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:56]:
```suggestion
## Delete the billing agreement
```

File: src/sales/billing-agreements.md [hunk 7]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+_Billing agreement general information_
+
+## Related Orders
+
+Contains the list of the orders placed using current Billing Agreement
+
+![Billing Agreement View]({% link images/images/sales-billing-agreement-related-orders-tab.png %}){: .zoom}
+_Billing agreement related orders tab_
+
+## Cancel the Billing Agreement
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:44]:
```suggestion
## Cancel the billing agreement
```

File: src/sales/billing-agreements.md [hunk 8]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. On the _Admin_ sidebar, go to  **Sales** > _Operations_ > **Billing Agreements**.
+
+1. Find the billing agreement in the list.
+
+1. Click the needed billing agreement to be viewed.
+
+1. In the right upper corner click the **Cancel** button.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:52]:
```suggestion
1. In the top-right corner, click **Cancel**.
```

File: src/sales/billing-agreements.md [hunk 9]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+1. To confirm the action click **OK**.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:54]:
```suggestion
1. To confirm the action, click **OK**.
```

File: src/sales/billing-agreements.md [hunk 10]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+- Status: Payment agreement status.
+- Created At: Creation date.
+- Updated At: Update dae.
+
+![Billing Agreement View]({% link images/images/sales-billing-agreement-view.png %}){: .zoom}
+_Billing agreement general information_
+
+## Related Orders
+
+Contains the list of the orders placed using current Billing Agreement
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:39]:
```suggestion
This tab displays the list of the orders placed using the current billing agreement.
```

File: src/sales/billing-agreements.md [hunk 11]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
 
 1. Find the billing agreement in the list.
 
-1. In the _Actions_ column, click **View**.
+1. Click the needed billing agreement to be viewed.
 
-### Column descriptions
+## Billing Agreement View
+
+Each Billing Agreement page consist of two tabs: **General Information** and **Related Orders**.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:22]:
```suggestion
Each billing agreement page consists of two tabs: _General Information_ and _Related Orders_.
```

File: src/sales/billing-agreements.md [hunk 12]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
...
+
+## General Information
+
+Contains general information about billing agreement:
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:26]:
```suggestion
This tab includes the general information about the billing agreement:
```

File: src/sales/billing-agreements.md [hunk 13]
```diff
@@ -15,15 +15,64 @@ _Billing agreements_
 
 1. Find the billing agreement in the list.
 
-1. In the _Actions_ column, click **View**.
+1. Click the needed billing agreement to be viewed.
```
Comment (leslietilling) [inline src/sales/billing-agreements.md:18]: Combine these steps to one like the other following procedures:

`1. Find the billing agreement in the list and click to open it.`

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits for style and clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**